### DISCUSSION TITLE
Gatsby Cloud, hosting two different branches from GitHub

### DISCUSSION THREAD

**Discussion by MrFacundo** [gatsbyjs/gatsby#36797] [Help]
I'm working on a Gatsby project and I want to deploy the current version of the site which is in development.

So far, I've deployed the site from a GitHub repository, the host is Gatsby Cloud. I have the hosting configured so that a branch is being deployed. This branch is just a page saying that the website is being built. This deployment is accesed through a custom domain.

I want to keep this version on the same domain it is right now. But I also would like to host a development version in any other domain, such as test.customdomain.com or customdomain.com/development, or anything else.

Is this possible using Gatsby Cloud ? I couldn't find this option on the platform

tldr: Using Gatsby Cloud, I want to host two different branches of my project during development, "branch1" on main domain and "main" on subdomain
